src/jit/scopeinfo.cpp

```diff
@@ -999,12 +999,12 @@ void                CodeGen::psiBegProlog()
             {
 #ifdef DEBUG
                 var_types regType = compiler->mangleVarArgsType(lclVarDsc1->TypeGet());
-#ifdef _TARGET_ARM_
+#ifdef FEATURE_HFA
```

Comment:
This #ifdef can be removed since now we are using accessor lvIsHfaRegArg() that returns false for non-hfa targets.